shell/config_helpers.go

```diff
@@ -1,17 +1,16 @@
-package config
+package shell
 
 import (
 	"fmt"
 	"path/filepath"
 	"regexp"
 	"strings"
 
-	"github.com/gruntwork-io/terragrunt/shell"
-
 	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/sts"
 	"github.com/gruntwork-io/terragrunt/errors"
+	. "github.com/gruntwork-io/terragrunt/models"
```

Comment:
I needed to refactor some type declarations and move them to new or other packages, thus I had to add new import declarations. The refactoring was required because the GO compiler detected loops in the imports and in contrast to other languages GO is not able to resolve such loops by itself. To limit the required code changes inside the affected code files I added the period in front of the new import statements, because according to the go language specification

> "If an explicit period (.) appears instead of a name, all the package's exported identifiers declared in that package's package block will be declared in the importing source file's file block and must be accessed without a qualifier."

https://golang.org/ref/spec#Import_declarations